### PR TITLE
#161717042 view articles authored by the user

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do?

Allow authenticated users that are authors view their articles
#### Description of Task to be completed?

Currently, our application only returns all articles

This task allows us to return articles for an authenticated user

#### How should this be manually tested?

Run the server `./manage.py runserver`
On postman create articles using different users
Using their respective tokens navigate to `http://127.0.0.1:8000/api/article/my-articles/`

#### What are the relevant pivotal tracker stories?

[#161717042](https://www.pivotaltracker.com/story/show/161717042)

#### Any background context you want to add?
N/A

#### Screenshots
image

### CODE REVIEW

**Comment by MuhanguziDavid** [comment]
Nice work @mendozabree , however, you may want to ensure that tests are passing.

File: authors/apps/articles/views.py [hunk 1]
```diff
@@ -48,6 +48,19 @@
...
+class ReturnArticle(APIView):
+    renderer_classes = (ArticleJSONRenderer,)
+    serializer_class = ArticleSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        # import pdb; pdb.set_trace()
+        articles = Article.objects.filter(author_id=request.user.id)
+        serializer = self.serializer_class(articles, many=True)
+        # print("******"+articles+"*********")
```
Comment (RonKbS) [inline authors/apps/articles/views.py]: Nice work @mendozabree.
Please remove this print and the pdb.

Comment (mendozabree) [reply authors/apps/articles/views.py]: On it thanks @RonKbS

**Review by kakaemma — APPROVED** [review]
LGTM

**Review by RonKbS — APPROVED** [review]
LGTM

**Review by samuelbwambale — APPROVED** [review]
LGTM

**Pull request merged**